### PR TITLE
Feature/25 adds docker compose

### LINKED ISSUE #25: Adds docker-compose for postgreSQL

### PR DESCRIPTION
**Pull request by csiszaralex**
Added docker-compose and also modified .env and environments.ts

### CODE REVIEW

File: packages/backend/src/util/environment.ts [hunk 1]
```diff
@@ -1,3 +1,13 @@
-import * as env from 'env-var'
+import { config } from 'dotenv'
+import { from } from 'env-var'
+const env = from(process.env, {})
+config()
 
-export const BACKEND_PORT = env.get('BACKEND_PORT').default(3300).asInt()
```
Comment (KostyalBalint) [inline packages/backend/src/util/environment.ts]: Same as in #34

**Review by KostyalBalint — APPROVED** [review]
lgtm
This contains the same changes as in #34, why to make 2 PR-s for this?

File: packages/backend/src/util/environment.ts (hunk 1 above)
Comment (csiszaralex) [reply packages/backend/src/util/environment.ts]: This branch based on branch used ín #34 I dont know why or how happend

Comment (KostyalBalint) [reply packages/backend/src/util/environment.ts]: Cause you oppend up the PR against the main, and that PR also not merged yet. So the changes against the main still apper. If it's based on that branch, I don't think a conflict will appear, but be careful when merging

Comment (csiszaralex) [reply packages/backend/src/util/environment.ts]: Ok thanks for advice

**Pull request merged**